src/main/java/org/sagebionetworks/bridge/worker/ExportWorker.java

```diff
@@ -0,0 +1,69 @@
+package org.sagebionetworks.bridge.worker;
+
+import java.util.concurrent.LinkedBlockingQueue;
+
+/**
+ * Generic export worker. This class encapsulates basic configuration, helpers, and clients needed by export workers
+ * and serves as the parent class for both Synapse export workers and iOS survey export workers.
+ */
+public abstract class ExportWorker extends Thread {
+    // Configured externally
+    private ExportWorkerManager manager;
+    private String studyId;
+
+    // Internal state
+    private final LinkedBlockingQueue<ExportTask> taskQueue = new LinkedBlockingQueue<>();
```

Comment:
Right now, the health data record table is only 800mb for 3 weeks worth of data, so I don't think memory will be a concern. For scaling 100x, we should consider moving the task queue to SQS. I've also included this in JIRA https://sagebionetworks.jira.com/browse/BRIDGE-458